src/sql/workbench/contrib/notebook/browser/notebookLinkHandler.ts

```diff
@@ -53,11 +52,13 @@ export class NotebookLinkHandler {
 			this.isMarkdown = this._link.attributes['is-markdown']?.nodeValue === 'true' ? true : false;
 			this.isEncoded = this._link.attributes['is-encoded']?.nodeValue === 'true' ? true : false;
 			this._isFile = this._link.protocol === `${Schemas.file}:` || this._link.protocol === `${Schemas.vscodeFileResource}:`;
-			// Given an anchor element for windows href link will need to use nodeValue instead as that does not encode the url
+			// When editing on richtext, for local files paths the nodeValue results are ambiguous depending on OS
```

Comment:
Add link to any documentation or bug reports on this - it seems like a bug and we want to be sure to have as much context on why we need to do this here in case it gets fixed in a later Node.JS update or something...

Reply:
Couldn't find any issues/documentation, I'm not sure where I stumbled upon the normalize fix. Will keep looking and update it if can find it.